### DISCUSSION TITLE
Error "No Unix_args"

### DISCUSSION THREAD

**Discussion by Liicha16** [thecoder-001/MineColab#98] [General]
Hi!  I have a problem when starting my server in Forge 1.8.9 in which it tells me this:
No unix_args.txt found.
Starting server...
No unix_args.txt found.

I need help because, I would like to have a server hosted this way for the moment until I get something better.
Thanks!
![MineColabMichasX-ipynb-Colaboratory](https://github.com/thecoder-001/MineColab/assets/158114267/4e0977c3-836a-4ff5-831b-20a432c72038)

**Comment by N-aksif-N** [comment]
Hello, If you don't might, try this: https://github.com/N-aksif-N/MineColab_Improved
It's a improvement for minecolab
